app/options.html

```diff
@@ -32,7 +32,7 @@
         <input type="checkbox" id="useStartDate"><br />
         <div id="startDateShowHideWrapper" style="display:none">
             <span>Format: yyyy-MM-dd, e.g. 2012-12-31</span><br />
```

Comment:
Do we bin off the formatting line now?

Reply:
You can still manually input so my UX head says no

Reply:
My concern is that Chrome shows the date as "dd/MM/yyyy" - it's inconsistent. 
Currently it won't be supported on any other browser so don't need to worry about UA support of date input type. Maybe old Chrome versions?